<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>756</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>120</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>解算算法：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>2000/01/01/00/00/00</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>230</y>
      <width>461</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>110</y>
      <width>391</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0,0">
     <property name="spacing">
      <number>5</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="LSM_Button">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>2</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>2</height>
        </size>
       </property>
       <property name="text">
        <string>最小二乘</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="KALMAN_Button">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>2</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>2</height>
        </size>
       </property>
       <property name="text">
        <string>卡尔曼滤波</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="KALMANCVButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>2</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>2</height>
        </size>
       </property>
       <property name="text">
        <string>卡尔曼滤波CV</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="KALMANCAButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>2</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>2</height>
        </size>
       </property>
       <property name="text">
        <string>卡尔曼滤波CA</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="showimage">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>201</width>
      <height>471</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="_resource.qrc">:/new/lingmeng.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>190</y>
      <width>411</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>0</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Time Start(GPST)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>0</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Time End(GPST)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>2000/01/01/00/00/00</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>解算设置：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>180</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>解算进度：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Config_Button">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>60</y>
      <width>94</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>0</width>
      <height>2</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>2</height>
     </size>
    </property>
    <property name="text">
     <string>解算设置</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>756</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="daoru_O"/>
    <addaction name="daoru_P"/>
    <addaction name="separator"/>
    <addaction name="baocun"/>
    <addaction name="guanbi"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>软件信息</string>
    </property>
    <addaction name="action_2"/>
   </widget>
   <widget class="QMenu" name="menu_D">
    <property name="title">
     <string>绘图(&amp;D)</string>
    </property>
    <addaction name="Draw_X"/>
    <addaction name="Draw_Y"/>
    <addaction name="Draw_Z"/>
    <addaction name="Draw_X_Y"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_D"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="daoru_O">
   <property name="text">
    <string>导入O文件(&amp;O)</string>
   </property>
  </action>
  <action name="baocun">
   <property name="text">
    <string>保存文件(&amp;S)</string>
   </property>
  </action>
  <action name="guanbi">
   <property name="text">
    <string>关闭程序(&amp;C)</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>软件信息</string>
   </property>
  </action>
  <action name="Draw_X">
   <property name="text">
    <string>绘制X坐标图</string>
   </property>
  </action>
  <action name="Draw_Y">
   <property name="text">
    <string>绘制Y坐标图</string>
   </property>
  </action>
  <action name="Draw_Z">
   <property name="text">
    <string>绘制Z坐标图</string>
   </property>
  </action>
  <action name="daoru_P">
   <property name="text">
    <string>导入P文件(&amp;P)</string>
   </property>
  </action>
  <action name="Draw_X_Y">
   <property name="text">
    <string>绘制X-Y图</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="_resource.qrc"/>
 </resources>
 <connections/>
</ui>
